hpx/util/logging/format/destination/file.hpp

```diff
@@ -119,12 +118,12 @@ struct file_t : is_generic, non_const_context<detail::file_info>
 
         if (!non_const_context_base::context().out)
```

Comment:
http://cdash.cscs.ch//viewBuildError.php?buildid=45664

Reply:
Fixed. I had accidentally introduced the extra indentation during refactoring.

Reply:
Thanks!